common/shared.c

```diff
@@ -283,7 +283,8 @@ void ft_csusage(char *name, char *desc)
 
 	fprintf(stderr, "\nOptions:\n");
 	fprintf(stderr, "  -n <domain>\tdomain name\n");
-	fprintf(stderr, "  -p <port>\tnon default port number\n");
+	fprintf(stderr, "  -b <src_port>\tnon default source port number\n");
```

Comment:
Quoting @shefty from previous comments:
>Also, I would keep -p to mean the dst_port, which is the common case. For the source port, maybe use -b? (for bind port)

Using -b looks fine to me. Let me know what you think.